src/DataFactory/DataFactoryV2/IntegrationRuntimes/SetAzureDataFactoryIntegrationRuntimeCommand.cs

```diff
@@ -279,6 +279,51 @@ public class SetAzureDataFactoryIntegrationRuntimeCommand : IntegrationRuntimeCm
             IgnoreCase = true)]
         public string Edition { get; set; }
 
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByIntegrationRuntimeName,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyIntegrationRuntimeName)]
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByResourceId,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyIntegrationRuntimeName)]
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByIntegrationRuntimeObject,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyIntegrationRuntimeName)]
+        [ValidateNotNullOrEmpty]
+        public string DataProxyIntegrationRuntimeName { get; set; }
+
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByIntegrationRuntimeName,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyStagingLinkedServiceName)]
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByResourceId,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyStagingLinkedServiceName)]
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByIntegrationRuntimeObject,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyStagingLinkedServiceName)]
+        [ValidateNotNullOrEmpty]
+        public string DataProxyStagingLinkedServiceName { get; set; }
```

Comment:
Should we have a completer for this? Is this a name or a resource id, or something else?

Reply:
it's linked service name. A completer might work, but it need to list all linked service. Since linked service is created by customer, I think he should know it well